logging/logrus/interceptors_test.go

```diff
@@ -214,3 +218,82 @@ func (s *LogrusLoggingSuite) TestPingList_WithCustomTags() {
 	assert.Contains(s.T(), msgs[1], `"level": "info"`, "OK error codes must be logged on info level.")
 	assert.Contains(s.T(), msgs[1], `"grpc.time_ms":`, "interceptor log statement should contain execution time")
 }
+
+func TestLogrusLoggingOverrideSuite(t *testing.T) {
```

Comment:
do you need a new suite here?

How about just introduce an override function in tests that you can change in a particular test?